plugins/isecl/tools/setup-isecl.sh

```diff
@@ -0,0 +1,380 @@
+#!/bin/bash
+set -e
+
+
+###################################
+# Environment variables
+WORK_DIR=${WORKDIR:=$HOME}
+BINARIES_DIR={BINARIES_DIR:-$WORK_DIR/isecl-build/binaries}
+ISECL_RELEASE=${ISECL_RELEASE:-v4.2.0-Beta}
+SCS_RELEASE=${SCS_RELEASE-v4.1.2}
+SQVS_RELEASE=${SQVS_RELEASE:-v4.1.2}
+UTILS_RELEASE=${UTILS_RELEASE:-v4.2/develop}
+
+SYSTEM_IP:=${SYSTEM_IP:-$(ip -o route get to 1 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')}
+SYSTEM_SAN=${SYSTEM_SAN:-"localhost,$(hostname)"}
+
+INSTALL_ADMIN_USERNAME=${INSTALL_ADMIN_USERNAME:-"admin"}
+INSTALL_ADMIN_PASSWORD=${INSTALL_ADMIN_PASSWORD:-"admin"}
+
+AAS_PORT=${AAS_PORT:-8444}
+AAS_URL=https://$SYSTEM_IP:$AAS_PORT/aas/v1
+AAS_ADMIN_USERNAME=${AAS_ADMIN_USERNAME:-"admin@aas"}
+AAS_ADMIN_PASSWORD=${AAS_ADMIN_PASSWORD:-"admin"}
+AAS_DB_NAME=${AAS_DB_NAME:-"aasdb"}
+AAS_DB_USERNAME=${AAS_ADMIN_USERNAME:-"user@aasdb"}
+AAS_DB_PASSWORD:=${AAS_DB_PASSWORD:-"aasdbpwd"}
+
+CMS_PORT=${CMS_PORT:-8445}
+CMS_URL=https://$SYSTEM_IP:$CMS_PORT/cms/v1/
+
+KBS_PORT=9443
+KBS_SERVICE_USERNAME=${KBS_SERVICE_USERNAME:-"user@kub"}
+KBS_SERVICE_PASSWORD=${KBS_SERVICE_PASSWORD:-"kbspasswd"}
+
+SCS_PORT=${SCS_PORT:-9000}
+SCS_URL=https://$SYSTEM_IP:$SCS_PORT/scs/sgx/certification/v1
+SCS_ADMIN_USERNAME=${SCS_ADMIN_USERNAME:-"admin@scs"}
+SCS_ADMIN_PASSWORD=${SCS_ADMIN_PASSWORD:-"admin"}
```

Comment:
I think we shouldn't even try using the default password, it seems wrong (also for the three other passwords).

Reply:
This script is just a helper to install on a bare-metal machine outside the cluster. I was just trying to keep it simple to use.